client/src/App.js

```diff
@@ -15,6 +16,7 @@ function App() {
       <LoginButton />
       <LogoutButton />
       <Profile />
+      <Recipe/>
```

Comment:
Navigation for these will be complex as there is all these components will be rendered on the same page. Should initiate something like react-router so that we can have separate tabs for different forms of data.